include/libtorrent/thread_pool.hpp

```diff
@@ -59,8 +59,8 @@ namespace libtorrent
 				while (m_num_threads < n)
 				{
 					++m_num_threads;
-					m_threads.push_back(boost::shared_ptr<thread>(
-						new thread(boost::bind(&thread_pool::thread_fun, this, int(m_num_threads)-1))));
+					m_threads.emplace_back(
```

Comment:
oh uh. this will conflict with a patch I've been baking today. I'm just putting the threads straight into the vector instead, since we have move semantics now